tutorials/io/binary_serialization_api.rst

```diff
@@ -385,8 +385,29 @@ Every name string is padded to 4 bytes.
 16: :ref:`RID<class_rid>` (unsupported)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-17: :ref:`Object<class_object>` (unsupported)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+17: :ref:`Object<class_object>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++----------+-------+-----------+---------------------------------------------------------------------+
+| Offset   | Len   | Type      | Description                                                         |
++==========+=======+===========+=====================================================================+
+| 4        | 4     | Integer   | val&0x7FFFFFFF = properties, val&0x80000000 = shared (bool)         |
++----------+-------+-----------+---------------------------------------------------------------------+
+
+Then what follows is, for amount of "properties", pairs of name and value,
+one after the other, using this same format.
+
+.. Note::
+
+   Only properties that are configured as storable will be included when
+   serializing an object. By default, script variables are configured as
+   :ref:`PROPERTY_USAGE_SCRIPT_VARIABLE<class_@GlobalScope_constant_PROPERTY_USAGE_SCRIPT_VARIABLE>`,
+   which is **not** included when serializing an object.
```

Comment:
> only properties that are configured as storable will be included

It's not clear what "configured as storable" means.

> By default, script variables are configured as [constant PROPERTY_USAGE_SCRIPT_VARIABLE], which is [b]not[/b] included.


Note that for example exported property like export var something has more flags set, including PROPERTY_USAGE_STORAGE and thus it would be stored.

I'm not sure what flags are set for what properties but for sure info added in here needs to be correct.

Reply:
I've re-worded this to hopefully be a little clearer.

And I re-worded it to say "By default, non-exported script variables are configured as PROPERTY_USAGE_SCRIPT_VARIABLE", which I think is correct, but I could instead re-word this as something more general, like "For example, script variables are often configured as PROPERTY_USAGE_SCRIPT_VARIABLE and are not included."

Reply:
I added the same latest fix here as in the other PR.